Cartfile.private

```diff
@@ -1 +1,2 @@
 github "facebook/ios-snapshot-test-case"
+github "erikdoe/ocmock"
```

Comment:
oh weird, this line disappeared?

Reply:
Yeah I don't know what happened. It was still working with bootstrap cause it was in Cartfile.resolved